<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EthScheduler</class>
 <widget class="QMainWindow" name="EthScheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="worker_tableWidget">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_worker_pushButton">
      <property name="text">
       <string>Start Worker</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_worker_pushButton">
      <property name="text">
       <string>Delete Worker</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEthScheduler">
    <property name="title">
     <string>EthScheduler</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Worker"/>
   </widget>
   <addaction name="menuEthScheduler"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAdd_Worker">
   <property name="text">
    <string>Add Worker</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
